src/book/03-guides/01-main-codebase/02-maintaining-subsystems.mdx

```diff
@@ -48,12 +48,50 @@ The resulting network should be exported to a [yaml](https://yaml.org/) file and
 
 3. Add this configuration file to the NUbots repository in the [VisualMesh module](https://github.com/NUbots/NUbots/tree/main/module/vision/VisualMesh/data/config/networks). Replace or add a configuration file depending on the use case of the Mesh - `RobocupNetwork.yaml` is for soccer playing on the real robot and `WebotsNetwork` is for soccer playing in the Webots simulator. View the [Git Guide](/guides/general/git) for information on using Git and submitting this change in a pull request.
 
+#### Running OpenCL not on the robot
+
+The Visual Mesh works best running the OpenCL framework, however most operating systems are not set up to run this by default. Whilst running the Visual Mesh on CPU is possible, sometimes it is best to replicate an environment as close to what is running on the robot as possible. Follow the steps below to set up OpenCL on a device other than a robot.
+
```

Comment:
I need to confirm that these steps are correct with a fresh install so general review is fine but these steps may change